lib/verifiablePresentation/SSI.types.ts

```diff
@@ -21,14 +18,14 @@ export interface CredentialStatus {
 export interface Credential {
   '@context': string[];
   id: string;
-  type: string[];
-  credentialSubject: CredentialSubject;
-  issuer: string;
+  type: string[] | string;
+  credentialSubject: unknown;
```

Comment:
Hmzz, I don't think I like the change away from CredentialSubject to unknown. Effectively the previous version already did that and the CredentialSubject should have an optional id for instance